### PR TITLE
DOCS-1947-CE-3.17.3

### PR DESCRIPTION
**Pull request by bmckercher123**
Patch release.

Product Version(s):
CE 3.17.3

Issue:
https://tigera.atlassian.net/browse/DOCS-1947

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx [hunk 1]
```diff
@@ -94,7 +94,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | MKE version                          | {{prodname}} support                 | Kubernetes versions |
 | -------------------- | ------------------------------------ | ------------------------------------ | ------------------- |
-| 3.17                 | MKE 3.6                              | {{prodname}} CNI with network policy | 1.24                |
+| 3.17                 | MKE 3.7.1                            | {{prodname}} CNI with network policy | 1.24                |
```
Comment (sabags) [inline calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx]:
```suggestion
| 3.17                 | MKE 3.7.1                            | {{prodname}} CNI with network policy | 1.27                |
```

Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise_versioned_docs/version-3.17/releases.json [hunk 2]
```diff
@@ -1,4 +1,271 @@
 [
+  {
+    "title": "v3.17.3",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "version": "v1.30.6",
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.17/releases.json]: v1.30.8

File: calico-enterprise_versioned_docs/version-3.17/releases.json [hunk 3]
```diff
@@ -1,4 +1,271 @@
...
+        "image": "tigera/honeypod-exp-service",
+        "version": "v3.17.3"
+      },
+      "honeypod-controller": {
+        "image": "tigera/honeypod-controller",
+        "version": "v3.17.3"
+      },
+      "key-cert-provisioner": {
+        "image": "tigera/key-cert-provisioner",
+        "version": "v1.1.13",
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.17/releases.json]: v1.1.14

File: calico-enterprise_versioned_docs/version-3.17/releases.json [hunk 4]
```diff
@@ -1,4 +1,271 @@
...
+      },
+      "eck-elasticsearch-operator": {
+        "version": "2.6.1"
+      },
+      "elasticsearch-operator": {
+        "image": "tigera/eck-operator",
+        "version": "v3.17.3"
+      },
+      "coreos-alertmanager": {
+        "version": "v0.25.0"
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.17/releases.json]: v0.25.1

**Review by danudey — DISMISSED** [review]
Just a few version updates and then lgtm

File: calico-enterprise_versioned_docs/version-3.17/releases.json (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.17/releases.json (hunk 3 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.17/releases.json (hunk 4 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/releases.json]: Fixed.

**Pull request merged**